src/js/components/Image/Image.js

```diff
@@ -1,46 +1,39 @@
-import React, { useState } from 'react';
-import { compose } from 'recompose';
-import { withForwardRef } from '../hocs';
+import React, { forwardRef, useState } from 'react';
 import { StyledImage } from './StyledImage';
 
-const Image = ({
-  a11yTitle,
-  fallback,
-  forwardRef,
-  onError,
-  opacity,
-  fill,
-  src,
-  ...rest
-}) => {
-  const [imageMissing, setImageMissing] = useState(false);
-  const handleError = event => {
-    if (onError) {
-      onError(event);
-    }
-    setImageMissing(true);
-  };
-  const extraProps = {
-    onError: (onError || fallback) && handleError,
-  };
-  return (
-    <StyledImage
-      aria-label={a11yTitle}
-      {...rest}
-      {...extraProps}
-      ref={forwardRef}
-      opacityProp={opacity}
-      fillProp={fill}
-      src={!imageMissing ? src : fallback}
-    />
-  );
-};
+const Image = forwardRef(
+  ({ a11yTitle, fallback, onError, opacity, fill, src, ...rest }, ref) => {
```

Comment:
I think image used to work as expected with with `ref`, why is that addition needed?

Reply:
Switching from `withForwardRef` HOC to `forwardRef` directly from React.